course-material/interactive-queries-and-parameterization/introduction-to-ipywidgets.md

```diff
@@ -1,15 +1,233 @@
 ---
-jupyter:
-  jupytext:
-    text_representation:
-      extension: .md
-      format_name: markdown
-      format_version: '1.3'
-      jupytext_version: 1.14.5
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.14.6
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
 ---
 
-# Introduction to ipywidgets
+# Introduction to `ipywidgets`
 
+```{code-cell} ipython3
+:tags: [remove-cell]
 
+# this cell is hidden in the docs, only used to simulate
+# the getpass() call
```

Comment:
Remove the getpass references

Also, not sure what the purpose of removing the cell is

Suggestion: remove this section from the notebook as it does not seem to be adding something and is a bit confusing to start here

Reply:
A lot of the linting issues are actually the same

`undefined name 'display'`

You can fix these via the import

```
from IPython.display import display
w = widgets.IntSlider()
display(w)
```

Reply:
Another thing I noticed is the subqueries file is in this PR
please remove it from this PR so that we don't create extra issues when handling the feedback from that file